enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -170,7 +172,19 @@
     // LDAP authorization settings
     //-----------------------------------------------------
 
-    @Description( "Perform LDAP search for authorization info using a system account." )
+    @Description( "Perform LDAP search for authorization info using a system account or the user's own account.\n\n" +
+                  "If this is set to `false` (default), the search for group membership will be performed " +
+                  "directly after authentication using the ldap context bound with the user's own account. " +
+                  "The mapped roles will be cached for the duration of `dbms.security.auth_cache_ttl`, " +
+                  "and then expire, requiring re-authentication. To avoid frequently having to re-authenticate " +
+                  "sessions you may want to set a relatively long auth cache expiration time together with this " +
+                  "option. NOTE: This option will only work if the users are permitted to search for the " +
+                  "group membership attributes on themselves in the directory.\n\n" +
+                  "If this is set to `true`, the search will be performed using a special system account user " +
+                  "with read access to all the users in the directory. " +
+                  "You need to specify the username and password with the next two settings below with his option. " +
```

Comment:
If "the next two settings" means making assumptions on order in the resulting content then indeed the two settings need to be referenced. Other components may have settings that are inserted in between, or the principle of order and grouping may change.